packages/api/src/middlewares/prividium-filtering.middleware.ts

```diff
@@ -0,0 +1,91 @@
+import { ForbiddenException, Injectable, NestMiddleware } from "@nestjs/common";
+import { Request, Response, NextFunction } from "express";
+import { AddressService } from "../address/address.service";
+import { LogService } from "../log/log.service";
+import { FilterTransactionsOptions } from "../transaction/transaction.service";
+import { getUrlWithoutParams } from "../common/utils";
+
+const OWNERSHIP_TRANSFERRED_TOPIC = "0x8be0079c531659141344cd1fd0a4f28419497f9722a3daafe3b4186f6b6457e0";
+
+const UNFILTERED_ROUTES = ["/auth", "/batches", "/blocks", "/health", "/ready", "/stats", "/tokens"];
```

Comment:
What's the difference of this and `UNPROTECTED_ROUTES` from `auth.middleware.ts`?